charts/gateway-helm/values.tmpl.yaml

```diff
@@ -18,6 +18,7 @@ global:
       pullPolicy: IfNotPresent
       # List of secrets in the same namespace of the component that can be used to pull images from private repositories.
       pullSecrets: []
+  extensionApis: {}
```

Comment:
can this be added in L97 instead ? under `config.envoyGateway` ?

Reply:
ok I've done that but I think it has unintended consequences. At least when I added all the stuff from `make generate` but still learning all this stuff